.github/workflows/ci.yml

```diff
@@ -66,6 +66,7 @@ jobs:
       run: |
         echo "Checking that code generation is up-to-date"
         make gen
+        make fmt
```

Comment:
would it be beneficial here to add an additional CI check to ensure the code has been `fmt'd` so we keep the repo in a clean state?

Reply:
Possibly, although I think the `lint` check already covers this because of https://github.com/hashicorp/consul-api-gateway/blob/main/.golangci.yml#L23-L25